pepagent/pepagent.py

```diff
@@ -468,3 +624,51 @@ def _check_conn_db(self) -> None:
         cols_name.sort()
         if DB_COLUMNS != cols_name:
             raise SchemaError
+
+
+def main():
+    # Create connection to db:
+    # projectDB = PepAgent(
+    #     user="postgres",
+    #     password="docker",
+    # )
+    projectDB = PepAgent("postgresql://postgres:docker@localhost:5432/pep-base-sql")
+
+    # prp_project2 = peppy.Project("/home/bnt4me/Virginia/pephub_db/sample_pep/amendments2/project_config.yaml")
```

Comment:
Maybe remove all this commented out code and put it into an "interactive testing" reference script or something?